Terminal.Gui/View/Layout/ViewLayout.cs

```diff
@@ -148,8 +145,7 @@ public virtual Point ScreenToFrame (in Point location)
     ///         <see cref="LayoutSubview(View, Size)"/> and <see cref="OnDrawContent(Rectangle)"/> methods to be called.
     ///     </para>
     ///     <para>
-    ///         Changing this property will cause <see cref="Frame"/> to be updated. If the new value is not of type
-    ///         <see cref="PosAbsolute"/> the <see cref="LayoutStyle"/> will change to <see cref="LayoutStyle.Computed"/>.
+    ///         Changing this property will cause <see cref="Frame"/> to be updated.
```

Comment:
This is an unrelated note for thoughts on future work:

It would probably be a good idea to implement INotifyPropertyChanged and react to that here, as well as the other stuff it already does.

Currently, we have more general events for the layout pipeline, which is cool for what they do, but not very useful when all I want to do is, for example, react to only one property changing.

I've already got that in my list of event-related stuff to do, but just pointing out here since I had the thought and this is a place that would be raised.